src/commcare_cloud/ansible/roles/postgresql_base/templates/postgresql.conf.j2

```diff
@@ -21,7 +21,9 @@ ssl = {% if postgresql_ssl_enabled %}true{% else %}false{% endif %}
 
 # RESOURCE USAGE (except WAL)
 
-shared_buffers = {{ postgresql_shared_buffers }}
+{% if postgresql_shared_buffers %}
```

Comment:
this if statement is confusing (and the other one for `postgresql_effective_cache_size`). I don't think it should still be there

Reply:
Removed the if conditions